domains/_github-pages-challenge-ImShyMike.shymike.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "ImShyMike",
+    "email": ""
```

Comment:
You need to add another form of contact (like discord, twitter etc.).

Reply:
done